contrib/k8s/matchbox-deployment.yaml

```diff
@@ -7,11 +7,15 @@ spec:
   strategy:
     rollingUpdate:
       maxUnavailable: 1
+  selector:
+      matchLabels:
+        app: matchbox
```

Comment:
match `name: matchbox` please

Reply:
Sounds good.  Does make the selector/labels a bit cleaner.